### DISCUSSION TITLE
My mastodon server is up and running but I can't use my new account to login to other servers.

### DISCUSSION THREAD

**Discussion by jessicaboggia** [mastodon/mastodon#21521] [General]
My instance is live at:
https://mastodon.godsibb.net and seems to work fine.

But I can't use the account I registered there to login to other servers like mastodon.social. Do I need to change a setting or something? I would be very grateful if someone could please point me in the right direction for whatever I am missing.

Thanks in advance!

**Comment by n33d4n4m3** [comment]
Hey, you didn't do anything wrong.
It is the concept of Mastodon that logging into other instances does not work. Each account is strictly bound to the instance on which the account was created. However, this doesn't mean that you can't interact with users from other instances with your account.
Just try to find out the @ of a person and type it into the search bar of your instance. You can follow the person, comment on their posts, etc. You can also configure relays. They make sure that your instance's timeline fills up with posts from other instances.

**Reply by jessicaboggia** [reply]
Thanks for clarifying @n33d4n4m3 ! 

I was able to successfully follow some outside instances, however I can only see their content a few hours later. When I first pull up a users page through my server, I can see it but it's empty. If I follow it, I can see it a few hours later. Is this also expected behavior? Sorry for asking so many noob questions, I'm finding this works a bit differently than I pictured it. I appreciate how nice you were in answering this question!! Thank you again.
